pkg/gateway/multiparts/tracker_test.go

```diff
@@ -73,6 +86,9 @@ func TestTracker_Get(t *testing.T) {
 				t.Errorf("Get() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
+			if !tt.wantErr {
+				got.CreationDate = got.CreationDate.UTC() // deepEqual workaround for time
```

Comment:
So the time is the only thing that could fail?
What if you expect a different error and everything unexpectedly works as expected, except for the `CreationDate` which is flaky?

Reply:
The CreationDate is not flaky it's just a different behavior between how it was saved originally in the DB and how protobuf's time object saves it.
protobuf's time object (timestamppb) doesn't save the timezone information so when read from the protobuf object we get the UTC time whereas the DB implementation when we read the timezone information, we get the time formatted using local timezone. The DeepEqual function has a "bug" where it compares two time objects without taking into account the timezone and this is simply a workaround to allow using DeepEqual and testing both implementations

Reply:
So why do it on the condition `!tt.wantErr`?

Reply:
Because if we got an error then got == nil and this line will fail

Reply:
Got it. The condition should actually be `(err == nil)` but the previous check makes it the same as `!tt.wantErr`. Consider changing to `(err == nil)` for clarity.

Reply:
Done